packages/vuetify/src/components/VSelect/VSelect.sass

```diff
@@ -128,3 +128,20 @@
       // the end of the container
       > input
         align-self: flex-end
+
+  &.v-input--dense
```

Comment:
`&.v-input.v-input--dense` to increase specificity and remove the `!important` below

Reply:
can you test it